src/scripts/main.js

```diff
@@ -1,3 +1,251 @@
 'use strict';
 
-// write your code here
+const score = document.querySelector('.game-score');
+let gameScore = 0;
+
+// сообщения в игре
+function gameMassedge(number) {
+  const messageCon = document.querySelectorAll('.message');
+
+  switch (number) {
+    case 0:
+      messageCon[0].className = messageCon[0].className.replace('hidden', '');
+      messageCon[1].className = messageCon[1].className + ' ' + 'hidden';
+      messageCon[2].className = messageCon[2].className + ' ' + 'hidden';
+
+      return;
+    case 1:
+      messageCon[0].className = messageCon[0].className + ' ' + 'hidden';
+      messageCon[1].className = messageCon[1].className.replace('hidden', '');
+      messageCon[2].className = messageCon[2].className + ' ' + 'hidden';
+
+      return;
+
+    default:
+      messageCon[0].className = messageCon[0].className + ' ' + 'hidden';
+      messageCon[1].className = messageCon[1].className + ' ' + 'hidden';
+      messageCon[2].className = messageCon[2].className + ' ' + 'hidden';
+  }
+}
+
+// рандомное значение
+function random() {
+  return Math.round(Math.random() * 15);
+}
+
+// значение 2 или 4 (10%)
+function meaning() {
```

Comment:
https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx
read about functions